src/engine/renderer/tr_image.cpp

```diff
@@ -2029,20 +2030,31 @@ static const multifileCubeMapFormat_t multifileCubeMapFormats[] =
 {
 	{
 		"OpenGL",
+		"_",
 		{ "px", "nx", "py", "ny", "pz", "nz" },
 		{ false, false, false, false, false, false },
 		{ false, false, false, false, false, false },
 		{ 0, 0, 0, 0, 0, 0 },
 	},
 	{
 		"Quake",
+		"_",
 		{ "rt", "lf", "bk", "ft", "up", "dn" },
 		{ true, true, false, true, true, false },
 		{ false, false, true, false, false, true },
 		{ 90, -90, 0, 0, 90, -90 },
 	},
+	{
+		"DarkPlaces",
```

Comment:
I had no idea to name this, Xonotic use some OpenGL-like skyboxes without `_` separator. I may name it “OpenGL” too.